docs/getting-started-guides/ovirt.md

```diff
@@ -0,0 +1,39 @@
+## What is oVirt
+
+oVirt is a virtual datacenter manager that delivers powerful management of multiple virtual machines on multiple hosts. Using KVM and libvirt, oVirt can be installed on Fedora, CentOS, or Red Hat Enterprise Linux hosts to set up and manage your virtual data center.
+
+## oVirt Cloud Provider Deployment
+
+The oVirt cloud provider allows to easily discover and automatically add new VM instances as nodes to you kubernetes cluster.
+At the moment there are no community-supported or pre-loaded images including kubernetes but it is possible to import or install Project Atomic in a VM to generate a template. Any other distribution that includes kubernetes may work as well.
```

Comment:
Can you clarify what you mean by images? OS/VM images or Docker images?

What does it mean to "generate a template"? What's it a template of or for?